pkg/validators/readiness/readiness.go

```diff
@@ -61,38 +68,34 @@ func (v *ReadinessValidator) Validate(ctx context.Context, resources []unstructu
 		if err != nil {
 			msg := fmt.Sprintf("could not determine readiness of resource Kind: %s Name: %s Namespace: %s",
 				resource.GetKind(), resource.GetName(), resource.GetNamespace())
-			v.logger.V(0).Info(msg, "error", err)
+			v.logger.Error(err, msg)
 			continue
```

Comment:
Please append the errors with `errors.Join()` and return the error

Reply:
Done